Upstream-Status: Inappropriate [configuration]

'tic' of some linux distributions (e.g. fedora 11) hang in an infinite
loop when processing the original file.

Index: ncurses-5.7/misc/terminfo.src
===================================================================
--- ncurses-5.7.orig/misc/terminfo.src
+++ ncurses-5.7/misc/terminfo.src
@@ -3706,12 +3706,11 @@ konsole-xf3x|KDE console window with key
 # The value for kbs reflects local customization rather than the settings used
 # for XFree86 xterm.
 konsole-xf4x|KDE console window with keyboard for XFree86 4.x xterm,
-	kend=\EOF, khome=\EOH, use=konsole+pcfkeys,
-	use=konsole-vt100,
-# Konsole does not implement shifted cursor-keys.
-konsole+pcfkeys|konsole subset of xterm+pcfkeys,
-	kLFT@, kRIT@, kcbt=\E[Z, kind@, kri@, kDN@, kUP@, use=xterm+pcc2,
-	use=xterm+pcf0,
+	kend=\EOF, kf1=\EOP, kf13=\EO2P, kf14=\EO2Q, kf15=\EO2R,
+	kf16=\EO2S, kf17=\E[15;2~, kf18=\E[17;2~, kf19=\E[18;2~,
+	kf2=\EOQ, kf20=\E[19;2~, kf21=\E[20;2~, kf22=\E[21;2~,
+	kf23=\E[23;2~, kf24=\E[24;2~, kf3=\EOR, kf4=\EOS,
+	khome=\EOH, use=konsole-vt100,
 # KDE's "vt100" keyboard has no relationship to any terminal that DEC made, but
 # it is still useful for deriving the other entries.
 konsole-vt100|KDE console window with vt100 (sic) keyboard,
